src/flags.rs

```diff
@@ -40,6 +50,26 @@ pub fn process(flags: &DenoFlags, usage_string: String) {
     log_level = log::LevelFilter::Debug;
   }
   log::set_max_level(log_level);
+
+  if flags.clear_cache {
+    if let Err(err) = deno_fs::remove_dir_contents_all(&state.dir.gen) {
+      eprintln!("Failed to clear cache: {}", err.description());
+      exit(1);
+    } else {
+      println!("Successfully cleared cache.");
+      exit(0);
+    }
+  }
+
+  if flags.clear_deps {
+    if let Err(err) = deno_fs::remove_dir_contents_all(&state.dir.deps) {
+      eprintln!("Failed to clear dependencies: {}", err.description());
+      exit(1);
+    } else {
+      println!("Successfully cleared dependencies.");
```

Comment:
maybe ought to be `debug!`?
I'd probably say `Successfully cleared dependencies: {}, &state.dir.deps)`

Reply:
`debug!` prints only when the user provides `-D`. I prefer to let the user know the effect of the flag.
Also I don't think `state.dir.deps` is an important info that should be exposed to the user.

Reply:
They know the effect of the flag, they passed it!